src/navigation/checkout/OrderTracking.js

```diff
@@ -101,7 +106,7 @@ class OrderTrackingPage extends Component {
       moment(order.shippingTimeRange[1]).format('HH:mm'),
   ]
 
-    const step = stateToStep(order.state)
+    const step = stateToStep(order.events)
```

Comment:
The problem here is that if the server does not return this property, it will break. It should be backwards compatible.